webapp/channels/src/components/widgets/users/avatar/avatar.tsx

```diff
@@ -70,7 +70,6 @@ const Avatar = ({
 
     return (
         <img
-            tabIndex={0}
```

Comment:
Do we not want this to be focusable any more? I can't tell if this was removed because of my previous comment

Reply:
Yes this is in response to removing the focus, since its handled by ProfilePopover. Checked the focus it works as expected as well

Reply:
Nope it will be focusable by the parent